src/bioregistry/analysis/paper_ranking.py

```diff
@@ -477,13 +479,13 @@ def runner(
     # These have already been curated and will therefore be filtered out
     curated_pubmed_ids: set[str] = {str(pubmed) for pubmed in df["pubmed"] if pd.notna(pubmed)}
 
-    # FIXME the fetch_pubmed_papers function should
-    #  take into account the start and end date. as
-    predictions_df = fetch_pubmed_papers(pubmed_ids_to_filter=curated_pubmed_ids, relative_date=30)
+    predictions_df = fetch_pubmed_papers(
+        pubmed_ids_to_filter=curated_pubmed_ids,
+        mindate=start_date,
```

Comment:
please use more pythonic names through the Bioregisty's API

```suggestion
        start_date=start_date,
```

you just need to have the _get_id function make the rename to fit with the pubmed API